projects/onto-search/src/lib/onto-search-facet/onto-search-facet.component.html

```diff
@@ -0,0 +1,20 @@
+<div *ngIf="type === BASIC">
+  <onto-basic-facet
+    [data]="data"
+    [type]="type"
+    [facetTitleTemplate]="customFacetTitleTemplate || facetTitleTemplate"
```

Comment:
What's the purpose of the basicFacetTemplate and facetTitleTemplate? Arent these mandatory for each facet type? I think these are redundant. @yordanalexandrov WDYT?

Reply:
removed